aguda.akinwunmi-AkinAguda/main.js

```diff
@@ -0,0 +1,156 @@
+var nav = document.getElementById('nav');
+var mask = document.getElementById("mask");
+var links = document.getElementsByClassName('item');
+var img = document.getElementsByTagName("header");
+var shouldSlide = true;
+var startSlide = false;
+nav.style.width = '0%';
+//Mobile Sliding Navigation
+function animHam(x) {
+    x.classList.toggle("change");
+    if(shouldSlide){
+        if(window.matchMedia("(min-width : 640px)").matches){
+            for(i = 0; i < links.length; i++){
+                links[i].style.visibility = "visible";
+            }
+            nav.style.width = '35%';
+        }
+        else{
+            for(i = 0; i < links.length; i++){
+                links[i].style.visibility = "visible";
+            }
+            nav.style.width = '94%';
+        }
+        shouldSlide = false;
+    }
+    else{
+        shouldSlide = true;
+        nav.style.width = '0%';
+        for(i = 0; i < links.length; i++){
+            links[i].style.visibility = "hidden";
+        }
+    }
+}
+//Animated Top Menu Bar
+window.onscroll = function(){headerAnim()}
+function headerAnim(){
+    if(document.body.scrollTop > 50 || document.documentElement.scrollTop > 50){
+        document.getElementById("top").classList.add("topAnim");
+        document.getElementById("line").classList.add("lineVanish")
+    }
+    else{
+        document.getElementById("top").classList.remove("topAnim");
+        document.getElementById("line").classList.remove("lineVanish")
+    }
+}
+function checkMedia(){
+    if(window.matchMedia("(min-width : 768px)").matches){
+        document.getElementById("top").appendChild(nav);
+    }
+    else{
+        document.getElementById("navigation").appendChild(nav);
+    }
+}
+//flashing images
+// var images = ['url("images/stocks.jpg")', 'url("images/streets.jpg")', 'url("images/tech.jpg")']
+// var i=1;
+// function switchImage(){
+//     checkMedia();
+//         setInterval(function(){
+//                 img[0].classList.add("flash");
+//                 setTimeout(function(){
+//                     img[0].classList.remove("flash");
+//                 },600)
+//                 setTimeout(function(){
+//                     img[0].style.backgroundImage = images[i];
+//                 },550)
+//                  i++;
+//                  if(i == images.length){
+//                     i = 0;
+//                  }
+//             },5000)
+// }
+window.onresize = checkMedia();
```

Comment:
Your checkMedia function would not be called on window resize. You should do
```js
window.onresize = checkMedia
```
or 
```js
window.onresize = e => checkMedia(e)
```

When the window is resized it would called `checkMedia` which is a function `checkMedia()` on the other hand is void